client/lib/wp/sync-handler/index.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Module dependencies
+ */
+import config from 'config';
+import localforage from 'localforage';
+import Hashes from 'jshashes';
+import { isWhitelisted } from './whitelist-handler';
+
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:sync-handler' );
+
+// defaults
+const defaults = config( 'sync-handler-defaults' );
+defaults.driver = localforage[ defaults.driver ];
```

Comment:
As long as I can see _localforage_ has its own key-definition for each driver: https://github.com/mozilla/localForage#configuration

```js
console.log( localforage.INDEXEDDB ); // -> asyncStorage
```
So we don't wanna set `asyncStorage` in development.json config file IMO.